setup.py

```diff
@@ -0,0 +1,29 @@
+
+
+import os
+
+from setuptools import setup, find_packages
+
+from heroprotocol import __version__
+
+
+setup(
+    name='heroprotocol',
+    version=__version__,
+    author='Blizzard Entertainment',
+    author_email='???',
```

Comment:
Unsure who the original author is, if it should just be Blizzard the company, and who the author email should be. Happy to remove if Blizzard doesn't want to supply one.